utils/Fetcher.ts

```diff
@@ -0,0 +1,11 @@
+import axios from "axios";
+import { Fetcher } from "swr";
+
+const _getData = async (arg: string) => {
+  const data = await axios.get(arg).then((res) => res.data);
+  return data.upvotes.toString();
+};
+
```

Comment:
can we make this into one whole function, theres no point in a "private" function if its just going to be exported.

Reply:
wdym